pages/common/conan-frogarian.md

```diff
@@ -0,0 +1,8 @@
+# conan frogarian
+
+> Displays the conan frogarian.
+> More information: <https://github.com/conan-io/conan/pull/6760>.
```

Comment:
Is there any documentation on the subcommand we can reference instead? Referencing the pull request seems rathe  unhelpful to me.

Reply:
I completely agree @sbrl, but it's practically a mystery!  There's not a single reference to it on the conan docs, and [the PR](https://github.com/conan-io/conan/pull/6760)'s description reads:
```
Changelog: omit
Docs: omit
```

>No idea what a frogarian is, but clearly it's useful somehow useful :P

It's an image of a frog with a sword.  The [conan docs 404 page](https://docs.conan.io/404) has it in more detail :)

It's arguable whether this page is even necessary, but I decided to go for it for completeness.

Reply:
Oh, I see! If there isn't a specific docs page for this subcommand, let's link to the main docs or the main GitHub repository.

Reply:
Okay.  I'll do that.